protocols/68ce98/68ce98.ot2.apiv2.py

```diff
@@ -0,0 +1,49 @@
+metadata = {
+    'protocolName': 'Custom Sample Transfer',
+    'author': 'Sakib <sakib.hossain@opentrons.com>',
+    'description': 'Custom Protocol Request',
+    'apiLevel': '2.8'
+}
+
+
+def run(ctx):
+
+    [asp_speed, disp_speed, asp_height,
+        disp_height] = get_values(  # noqa: F821
+        "asp_speed", "disp_speed", "asp_height", "disp_height")
+
+    asp_speed = float(asp_speed)
+    disp_speed = float(disp_speed)
+    asp_height = float(asp_speed)
+    disp_height = float(disp_speed)
+
+    # Load Labware
+    plate1 = ctx.load_labware('waters_96_wellplate_2ml', 1, 'Plate 1')
+    plate2 = ctx.load_labware('waters_96_wellplate_2ml', 2, 'Plate 2')
+    tiprack = ctx.load_labware('opentrons_96_tiprack_300ul', 4)
+
+    # Load Pipette
+    m300 = ctx.load_instrument('p300_multi_gen2', 'left', tip_racks=[tiprack])
+
+    # Get sample columns
+    plate1_wells = [plate1.columns()[i][0] for i in range(12)]
```

Comment:
You don't necessarily need to change this, but a simpler way to access the wells of a 96-well plate with the multi-channel could look like... `plate1_wells = plate1.rows()[0]`